integration_test/third_party_apps_test/applications/dcgm/debian_ubuntu/install

```diff
@@ -18,7 +18,22 @@ if [[ $ID == debian ]]; then
     sudo add-apt-repository contrib
 fi
 sudo apt update
-sudo apt -y install cuda 
+
+# TODO(b/357944360): confirm if the older GPUs can use >= 560 driver (CUDA 12.6
+# or newer) and then remove this case statement
+DEVICE_CODE=$(lspci -n | grep -Po '10de:[\w\d]{4}')
+case $DEVICE_CODE in
+    # V100 | P4 | P100
+    10de:1db1|10de:1bb3|10de:15f8)
```

Comment:
Is there some documentation of how we know which older GPUs are affected? And some doc pointing to how we identify them? Are these device codes a 1:1 with these GPUs?

Reply:
Done! Added links to the related docs. 

The device codes and the GPU variants are 1:1. E.g., on GCP there is only P100 16GB variant and it has one unique code for it. 

Re: which older GPUs are affected: the NVIDIA repo is ahead of the doc - driver 560 can be installed on Ubuntu now but its doc isn't available, so we couldn't be sure yet about the supported GPUs. Wait for the doc to confirm whether we actually need to have this device code check logic [b/357944360](http://b/357944360).